app/models/project.rb

```diff
@@ -33,7 +34,22 @@ class Project < ApplicationRecord
 
     validates :project_title, presence: true, length: { maximum: 255 }, if: :validate_title?
     validates :description, presence: true, if: :validate_description?
-    validate :validate_description_length, if: :validate_description?
+
+    validate do ||
```

Comment:
```suggestion
    validate do
```
I don't _think_ these pipes are doing anything.